config/redirects.js

```diff
@@ -2435,6 +2435,10 @@ module.exports = [
       from: '/connections/apple-setup',
       to: '/connections/apple-siwa/set-up-apple'
     },
+    {
+      from: '/connections/apple-siwa/add-siwa-web-app',
+      to: '/connections/social/apple'
+    },
```

Comment:
Going to sound like a broken record here, but more than one page moved right? Redirects for any page that moves ever.

Reply:
only one page moved

Reply:
`/connections/apple-siwa/add-siwa-to-web-app` was deleted, then remade here `/connections/social/apple` (redirect exists)

`connections/apple-siwa/add-siwa-to-native-app`  was moved to `/connections/nativesocial/add-siwa-to-native-app` (no redirect)

Reply:
I added it.